File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/BlockedUserQueryForm.java
package com.sdkwork.app.model;

public class BlockedUserQueryForm {
    private Integer page;
    private Integer size;
    private String keyword;
    private String startTime;
    private String endTime;

    public Integer getPage() {
        return this.page;
    }
    
    public void setPage(Integer page) {
        this.page = page;
    }

    public Integer getSize() {
        return this.size;
    }
    
    public void setSize(Integer size) {
        this.size = size;
    }

    public String getKeyword() {
        return this.keyword;
    }
    
    public void setKeyword(String keyword) {
        this.keyword = keyword;
    }

    public String getStartTime() {
        return this.startTime;
    }
    
    public void setStartTime(String startTime) {
        this.startTime = startTime;
    }

    public String getEndTime() {
        return this.endTime;
    }
    
    public void setEndTime(String endTime) {
        this.endTime = endTime;
    }
}

File: sdkwork-app-sdk-typescript/src/types/blocked-user-query-form.ts
/** 查询黑名单用户请求 */
export interface BlockedUserQueryForm {
  /** 页码 */
  page?: number;
  /** 每页大小 */
  size?: number;
  /** 关键词 */
  keyword?: string;
  /** 开始时间 */
  startTime?: string;
  /** 结束时间 */
  endTime?: string;
}
